webapp/package.json

```diff
@@ -7,11 +7,15 @@
   "main": "src/index.js",
```

Comment:
`package.json` in the starter template actually starts with:
```
{
  "private": true,
  "scripts": {
    "build": "webpack --mode=production",
...
```

Should we apply the same here?